backend/arm64/cfg_selection.ml

```diff
@@ -180,5 +180,5 @@ class selector =
   end
 
 let fundecl ~future_funcnames f =
-  Cfg.reset_next_instr_id ();
+  Cfg.reset_instr_id ();
```

Comment:
why do we need to reset here, instead of creating a fresh cfg?

Reply:
I did that to keep the pull request
obviously semantics-preserving,
but indeed I would like to eventually
revove that kind of state.